src/components/shared/chart/Canvas.js

```diff
@@ -115,6 +137,15 @@ export default class ChartCanvas<HoveredItem> extends React.Component<
     this._offsetY = event.nativeEvent.offsetY;
     const maybeHoveredItem = this.props.hitTest(this._offsetX, this._offsetY);
 
+    if (
+      Math.abs(this._offsetX - this._mouseDownOffsetX) >
+        MOUSE_CLICK_MAX_MOVEMENT_DELTA ||
+      Math.abs(this._offsetY - this._mouseDownOffsetY) >
+        MOUSE_CLICK_MAX_MOVEMENT_DELTA
+    ) {
+      this._mouseMoved = true;
+    }
```

Comment:
That was my original thought. But then one could do the mouse down, drag outside the bounds, but come back to the original region for the mouse up. In that case it would be interpreted as a click even though the mouse was moved a lot in between. Probably very unlikely, but I felt that this version was more correct :)

Reply:
OK, I see what you mean